src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -20,29 +19,20 @@ internal enum Stamp
             SetToMaxUsed = 3,
         }
 
-        internal int _version;
         private List<Cookie> _list = new List<Cookie>();
 
         private DateTime _timeStamp = DateTime.MinValue;
         private bool _hasOtherVersions;
 
-        private readonly bool _isReadOnly;
-
         public CookieCollection()
         {
-            _isReadOnly = true;
-        }
-
-        internal CookieCollection(bool IsReadOnly)
```

Comment:
GitHub and referencesource (which is .NET Framework 4.x, i.e. Desktop) are 2 places.  However, there is a third set of sources representing deltas with our UWP (.NET Native) versions of some of our libraries that differ from CoreCLR.  The .NET Native sources aren't yet on GitHub since we haven't opened sourced those yet.  

So, my point is that it is not necessarily obvious right now that something isn't being used by our stack.  So, we would need to manually verify this.